frontend/src/scenes/pipeline/Pipeline.tsx

```diff
@@ -19,13 +20,20 @@ import { Transformations } from './Transformations'
 
 export function Pipeline(): JSX.Element {
     const { currentTab, canEnableNewDestinations, canGloballyManagePlugins } = useValues(pipelineLogic)
+    const { hasEnabledImportApps } = useValues(importAppsLogic)
 
     let tabToContent: Partial<Record<PipelineTab, JSX.Element>> = {
         [PipelineTab.Overview]: <Overview />,
         [PipelineTab.Transformations]: <Transformations />,
         [PipelineTab.Destinations]: <Destinations />,
         [PipelineTab.SiteApps]: <FrontendApps />,
-        [PipelineTab.ImportApps]: <ImportApps />, // TODO: only show if some enabled
+    }
+    // Import apps are deprecated, we only show the tab if there are some still enabled
+    if (hasEnabledImportApps) {
+        tabToContent = {
+            ...tabToContent,
+            [PipelineTab.ImportApps]: <ImportApps />,
```

Comment:
Just one potentially annoying thing for users is that they won't know this only shows up if you have legacy sources enabled. If someone has one legacy source enabled and wants to temporarily disable it, they'll be in for a frustrating surprise.

I propose we just add a warning banner to the top of `ImportApps`, which would say "The sources below are being deprecated. You still have access, because you have at least one legacy source enabled." Ideally with a link to a blog post with the rationale and our plans, but not sure if that's in place

Reply:
Fine with them sending a support request at this point - all of them have clear plans for how the deprecation works & can be manually re-enabled in django admin.

Reply:
Fine with them sending a support request at this point - all of them have clear plans for how the deprecation works & can be manually re-enabled in django admin.